src/test/java/org/ice4j/ice/NetworkUtilsTest.java

```diff
@@ -23,6 +23,40 @@
 
 public class NetworkUtilsTest
 {
+    @Test
+    public void testIpv4StringToBytes()
+    {
+        byte[] addr = NetworkUtils.strToIPv4("1");
```

Comment:
I wonder if the first three assertions here represent a bug / unintended functionality. The last section of this test method asserts `Null` when an input string is invalid IPv4, but doesn't return `Null` for the first three assumptions here. 

The implication here is that "1", "1.2", and "1.2.3" are valid IPv4 addresses, which they are not. Is it worth flagging this to the repo-owner?

Reply:
I don't think this is a bug as the function takes in a string and generates a valid IPv4 address from it. That is a bit clearer (I think) now that I have used assertArrayEquals() elsewhere. The approach to all of the assertNull()'s that I added was to find a way to trigger another return null, and hence cover additional lines, by varying the input.